### PR TITLE
Minor improvements to the install documentation

### PR DESCRIPTION
**Pull request by stephane**

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -136,7 +152,13 @@ You also have to set where your tenant model is.
 
     TENANT_MODEL = "customers.Client" # app.Model
 
-Now run ``migrate_schemas --shared`` to create the shared apps on the ``public`` schema. Note: your database should be empty if this is the first time you're running this command.
+Now you can create your app migrations, ``customers`` in our case (otherwise Django >= 1.9 will not create its table):
```
Comment (goodtune) [inline docs/install.rst]: I suggest dropping the version 1.9 reasoning - the project only supports 1 out of 3 versions that would create the table without migrations; treat `makemigrations` as a necessary step.

File: docs/install.rst [hunk 2]
```diff
@@ -55,7 +56,26 @@ If you'd like to serve the public tenant for unrecognised hostnames instead, use
         #...
     )
 
-Make sure you have ``django.template.context_processors.request`` (``django.core.context_processors.request`` if you're on Django 1.8) listed under ``TEMPLATE_CONTEXT_PROCESSORS`` else the tenant will not be available on ``request``.
+Make sure you have ``django.template.context_processors.request`` listed under ``context_processors`` (Django 1.8+):
```
Comment (goodtune) [inline docs/install.rst]: Django versions older the 1.8 are unsupported - remove special case comments and any examples for unsupported versions along with it.

**Comment by bernardopires** [comment]
Ping @stephane, this is an awesome doc update, please fix!

**Comment by stephane** [comment]
@bernardopires done

File: docs/install.rst [hunk 3]
```diff
@@ -55,7 +56,24 @@ If you'd like to serve the public tenant for unrecognised hostnames instead, use
...
+                'context_processors': [
+                    # ...
+                    'django.template.context_processors.request',
+                    # ...
+                ]
+            }
+        }
+    ]
+
+If you are using an old version of Django (before v1.8), the following setting must be set in ``TEMPLATE_CONTEXT_PROCESSORS``:
```
Comment (goodtune) [inline docs/install.rst]: Older versions are unsupported, this is confusing.

Comment (stephane) [inline docs/install.rst]: Feel free to update my PR as you wish

**Pull request merged**